vnext/Desktop.DLL/react-native-win32.x86.def

```diff
@@ -25,7 +25,7 @@ EXPORTS
 ?InitializeTracing@react@facebook@@YGXPAUINativeTraceHandler@12@@Z
 ?Make@IWebSocketResource@React@Microsoft@@SG?AV?$shared_ptr@UIWebSocketResource@React@Microsoft@@@std@@$$QAV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@5@@Z
 ?Make@JSBigAbiString@react@facebook@@SG?AV?$unique_ptr@$$CBUJSBigAbiString@react@facebook@@U?$default_delete@$$CBUJSBigAbiString@react@facebook@@@std@@@std@@$$QAV?$unique_ptr@U?$IAbiArray@D@AbiSafe@@UAbiObjectDeleter@2@@5@@Z
-?MakeMemoryMappedBuffer@JSI@Microsoft@@YG?AV?$shared_ptr@VBuffer@jsi@facebook@@@std@@QB_WI@Z
+?MakeMemoryMappedBuffer@JSI@Microsoft@@YG?AV?$unique_ptr@VBuffer@jsi@facebook@@U?$default_delete@VBuffer@jsi@facebook@@@std@@@std@@QB_WI@Z
```

Comment:
Are we using this api in office?

Reply:
Not at all.
I checked before making this change instead of adding a new factory method.

Reply:
Can we remove remove it from the ABI then, or do we still need it for the  test project?

Reply:
Ideally, we should test it as an integration test (react-native-win32.dll), but other types used by the test are not in the ABI and we don't want to add them just for testing purposes.

TL;DR
We can remove it at least for now.